CGPs/0036/staging.json

```diff
@@ -0,0 +1,152 @@
+[
+    {
+      "contract": "Freezer",
+      "function": "freeze",
+      "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C"], // exchangeEUR address
+      "value": "0"
+    },
+    {
+        "contract": "Freezer",
+        "function": "freeze",
+        "args": ["0xcCA78ae41fe8F83eE7F37B0110eb6FAe3E9Ca0F0"], // stableTokenEUR address
+        "value": "0"
+      },
+
+
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x4D169C0Ec17D3dEcB250701Aa64e966bE98a696B"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x034217fB0C08dec29a6beA99e75aAC39baf7459A"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0xD20f803a1489849031725479Df27741c3326eD15"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x26412103865Be3F80152EA10E61E6294ed0e0DC9"], // stableTokenEUR address
+        "value": "0"
+      },
+      {
+        "contract": "SortedOracles",
+        "function": "addOracle",
+        "args": ["0x7Ba21D1f769Ae9feCF58dEfB5bD08905Bd3B707B"], // stableTokenEUR address
+        "value": "0"
+      },
+
+
+      // fund the oracles?
+      {
+        "contract": "GovernanceProxy",
+        "function": "",
+        "args": ["0x4D169C0Ec17D3dEcB250701Aa64e966bE98a696B"],
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "",
+        "args": ["0x034217fB0C08dec29a6beA99e75aAC39baf7459A"],
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "",
+        "args": ["0xD20f803a1489849031725479Df27741c3326eD15"],
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "",
+        "args": ["0x26412103865Be3F80152EA10E61E6294ed0e0DC9"],
+        "value": "0"
+      },
+      {
+        "contract": "GovernanceProxy",
+        "function": "",
+        "args": ["0x7Ba21D1f769Ae9feCF58dEfB5bD08905Bd3B707B"],
+        "value": "0"
+      },
+
+      // sent constitution parameters (for exchange) (same as the ones from governanceConstitution.js in monorepo)
+
+      {
+        "contract": "GovernanceProxy",
+        "function": "setConstitution",
+        "args": ["0x001E515b1902cAdAb233781eC73E9643B6c5012C", "", "0.9"], // setRegistry
```

Comment:
The arguments are the arguments in the `setConstitution` function:
https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/governance/Governance.sol#L417

So it would be the address of the (proxy) contract for which we are setting the constitution